Update Login page documentation (#40)
gatling/frontline-cloud-doc: content/reference/user/login/index.md

---
title: "Login"

> description: "Learn how to connect to Gatling Entreprise with Github."

Reviewer: ```suggestion
description: "Learn how to connect to Gatling Enterprise with GitHub."
```

Author: Done

date: 2021-03-10T09:29:04-05:00
lastmod: 2021-03-10T09:29:04-05:00
weight: 10010
---

To log in, you need a GitHub account.

If you want to connect with an existing organization, please reach out to a System Admin to [join an organization]({{< ref "../../admin/users#managing-users" >}}).

{{< img src="login-github.png" alt="Login button" >}}

Pressing the **Continue with GitHub** button will redirect you to GitHub, where you will be asked to authorize the application.

You need to **authorize** it. We only ask access to your public profile and email address.

{{< img src="login-github-oauth.png" alt="OAuth GitHub page" >}}

{{< alert danger >}}
The GitHub authorization page opens in a new tab. If no tab opens, make sure that your browser doesn't block it.
{{< /alert >}}

> If a System Admin added you to his organization, you're logged in.

Reviewer 2: ```suggestion
If a System Admin added you to their organization, you're logged in.
```

Author: Done

> weight: 10010

Author: Done

---

To log in, you need a GitHub account.

> If you want to connect with an existing organization, please reach out to your root administrator to [join an organization]({{< ref "../../admin/users#managing-users" >}}).

Reviewer 3: reach out to a System Admin

Author: Done

{{< img src="login-github.png" alt="Login button" >}}

Pressing the **Continue with GitHub** button will redirect you to GitHub, where you will be asked to authorize the application.

> You need to **authorize** it. We only ask access to your profile and email address.

Author: Done

{{< img src="login-github-oauth.png" alt="OAuth GitHub page" >}}

{{< alert danger >}}
The GitHub authorization page is opened in a new tab. If no tab opened, make sure that your browser didn't block it.
{{< /alert >}}

> You need to fill the Signup form to create an account on our application.
> 
> {{< img src="signup-gatling.png" alt="OAuth GitHub page" >}}
> 
> Pressing the **Create account** button to register you on our application. 

Reviewer 2: ```suggestion
{{< img src="signup-gatling.png" alt="OAuth GitHub page" >}}

To register your account, fill in the Signup form and click on the **Create account** button. 
```

Author: Done

> If a root administrator added you to his organization, you're logged in.

Reviewer 2: I don't think we show the signup form to a user if their account was created be a system admin, do we? So this isn't quite correct (the user won't even see a "create account" button in this case).

Reviewer 3: system admin, not root administrator

Author: Done

> Otherwise, you're redirected to the **Organization creation** page.

Reviewer 2: ```suggestion
Otherwise, you will be redirected to the **Organization creation** page.
```

Author: Done

{{< img src="organization-creation.png" alt="OAuth GitHub page" >}}

{{< alert danger >}}

> Creating an organization is irreversible. If you want to leave your organization, contact [support](https://gatlingcorp.atlassian.net/servicedesk/customer/portal/8).

Reviewer 3: Add a warning on they shouldn't create an organization if they want to join one

Author: Done
